SPECS/llvm/llvm.spec

```diff
@@ -37,8 +37,9 @@ export CXXFLAGS="`echo " %{build_cxxflags} " | sed 's/ -g//'`"
 mkdir -p build
 cd build
 cmake -G Ninja                              \
-      -DCMAKE_INSTALL_PREFIX=/usr           \
+      -DCMAKE_INSTALL_PREFIX=%{_prefix}           \
```

Comment:
Nit: could you also re-align the `\` signs, since they were so neatly done before? ;)